.github/workflows/python.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         working-directory: ./bindings/python
         run: |
           source .env/bin/activate
-          pip install black==20.8b1
+          pip install black==20.8b1 click==8.0.4
```

Comment:
https://github.com/psf/black/issues/2964

We could upgrade black too at some point.